modules/react/popup/stories/Popup.stories.mdx

```diff
@@ -0,0 +1,318 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Popup} from '@workday/canvas-kit-react/popup';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Simple} from './examples/Simple';
+import {InitialFocus} from './examples/InitialFocus';
+import {OpenWithoutModel} from './examples/OpenWithoutModel';
+import {ManualControl} from './examples/ManualControl';
+import {MultiplePopups} from './examples/MultiplePopups';
+import {NestedPopups} from './examples/NestedPopups';
+import {RTL} from './examples/RTL';
+import {UsePopup} from './examples/UsePopup';
+import {PopupModelConfigComponent, PopupStateComponent, PopupEventsComponent} from './PopupModel';
+
+<Meta title="Components/Popups/Popup/React" component={Popup} />
+
+# Canvas Kit Popups
+
+A "popup" is a classification for a type of stacked UI element that appears "on top" of statically
+positioned content. Tooltips, Modals, Dropdown menus, etc are all examples of "popups". Canvas Kit
+has a "stack manager" system for managing these stacked UIs. Different types of popups have
+different requirements of behavior for UX and accessibility - we can call them behaviors,
+capabilities, or traits. Canvas Kit comes with a number of [behaviors](#hooks) in the form of React
+Hooks.
```

Comment:
Nit: If the link is to a hooks section, the hooks reference should be included in the link

```suggestion
capabilities, or traits. Canvas Kit comes with a number of [behaviors in the form of React
Hooks](#hooks).
```

Reply:
What about "behavioral hooks"?